libavfilter/dnn/dnn_backend_tf.c

```diff
@@ -172,6 +172,24 @@ static DNNReturnType tf_start_inference(void *args)
     return DNN_SUCCESS;
 }
 
+/**
+ * Free the TFRequestItem completely.
+ *
+ * @param arg Address of the TFInferRequest instance.
+ */
+static inline void delete_request_item(TFRequestItem **arg) {
```

Comment:
for the function name, i don't see delete in ffmpeg, we may use free/destory

Reply:
Okay, I'll correct this.